projects/angular/src/forms/datepicker/day.ts

```diff
@@ -50,13 +52,20 @@ export class ClrDay {
   @Input('clrDayView')
   public set dayView(day: DayViewModel) {
     this._dayView = day;
-    this.dayString = this._dayView.dayModel.toDateString();
   }
 
   public get dayView(): DayViewModel {
     return this._dayView;
   }
 
+  public get dayString(): string {
+    return this.dayView.isSelected
```

Comment:
Why was this moved?

Reply:
The dayString is initialized with the input. And it works ok in the app because every time a date is selected the grid closes and when you open it, it initializes again, but the change detection on the unit test is not working this way and the test is not passing, that's why I moved it to getter. Maybe should be private or there is an other solution to this.

Reply:
Ah, okay.